common/constants/validations.js

```diff
@@ -1,2 +1,2 @@
 export const minimumPasswordLength = 8;
-export const passwordStrengthRegex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d).{8,}/;
+export const passwordStrengthRegex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d).+/;
```

Comment:
can you rename this one to `requiredCharactersRegex` per our chat?

Reply:
Yup sorry, thought we only meant the function but that makes sense too!